fast64_internal/oot/oot_level_writer.py

```diff
@@ -741,7 +743,21 @@ def ootProcessLOD(
     DLGroup.addDLCall(transparentLOD.draw, "Transparent")
 
 
-def ootProcessEmpties(scene, room, sceneObj, obj, transformMatrix):
+def getActorRotation(actorProp: OOTActorProperty, blendRots: list[int]):
+    # Figure out which rotation to export, Blender's or the override
+    override = "Override" if actorProp.actorID == "Custom" else ""
+    overrideRots = [getattr(actorProp, f"rot{override}{rot}") for rot in ["X", "Y", "Z"]]
+    exportRots = [f"DEG_TO_BINANG({(rot * (180 / 0x8000)):.3f})" for rot in blendRots]
```

Comment:
so there are three sets of rotations?
- one from the blender object
- two "override", one for non-custom actors and one for custom? why are these two different? why are the properties for "Custom" named "Override"?

Reply:
- blender object (which won't be used if the actor can have parameters in its rotation, which makes sense to me)
- string prop rotation for actors without the id "custom"
- string prop rotation for custom actors

iirc the override thing is gone and it all depend on the xml data, I fail to see how using rotations as real rotation and not parameter can be useful if the actor is using rotations

btw this depends on other parameters, some actors doesn't always actually check for the rotation for params, in this case the blender object rotation can be used (aka if you don't see any rotation param field in the fast64 ui)